docs/developer-docs/latest/developer-resources/database-apis-reference/rest-api.md

```diff
@@ -34,6 +34,8 @@ Use `qs` directly to generate complex queries instead of creating them manually.
 :::details Example using qs
 
 ```js
+// GET /api/books?sort[0]=title%3Aasc&filters[title][$eq]=hello&populate=%2A&fields[0]=title&pagination[pageSize]=10&pagination[page]=1&publicationState=live&locale[0]=en
```

Comment:
Shouldn't we move this outside the code-block to match others or is it here because it's too long?